web/content/3.0.md

```diff
@@ -6,6 +6,7 @@ path: "3.0"
 Available guides:
 
 * [Planning for Foreman](/{{< param "path" >}}/Planning_Guide/index-foreman-el.html)
+* [Foreman and Katello Release Notes](/{{< param "path" >}}/Release_notes/index-katello-el.html)
```

Comment:
I'd recommend linking to just Foreman. Also, the Katello flavor doesn't include `-el`.
```suggestion
* [Foreman Release Notes](/{{< param "path" >}}/Release_notes/index-foreman-el.html)
* [Foreman and Katello Release Notes](/{{< param "path" >}}/Release_notes/index-katello.html)
```

Reply:
Applied.